packages/field-base/src/text-area-slot-mixin.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import { dedupingMixin } from '@polymer/polymer/lib/utils/mixin.js';
+import { InputMixin } from './input-mixin.js';
+import { SlotMixin } from './slot-mixin.js';
+
+const TextAreaSlotMixinImplementation = (superclass) =>
+  class TextAreaSlotMixinClass extends InputMixin(SlotMixin(superclass)) {
+    get slots() {
+      return {
+        ...super.slots,
+        textarea: () => {
+          const native = document.createElement('textarea');
+          const value = this.getAttribute('value');
+          if (value) {
+            native.value = value;
```

Comment:
Hmm, not sure if `<textarea>` accepts the `value` attribute.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

> Default content entered between the opening and closing tags. <textarea> does not support the value attribute.

Reply:
The `attribute` is not supported, but the property **is** supported, so we set property here.